src/main/kotlin/app/revanced/util/resources/ResourceUtils.kt

```diff
@@ -25,9 +34,13 @@ object ResourceUtils {
             val key = attributes.getNamedItem("name")!!.nodeValue!!
             val value = it.textContent!!
 
-            val formatted = attributes.getNamedItem("formatted") == null
+            val formatted = !attributes.getNamedItem("formatted")?.nodeValue.equals("false", ignoreCase = true)
 
-            SettingsPatch.addString(key, value, formatted)
+            // Detect unescaped quotes that will throw generic AAPT errors
```

Comment:
Since AAPT is failing, the correct way to handle this case is to log the error output stream when executing the AAPT process. This way, any invalid resources would give proper feedback at compile time. But implementing that is not related to this PR which is why I was suggesting to remove this code, as the correct place to handle it would be outside the PR.

Reply:
> include the string but strip the quotes, and that cannot possible be what the developer wanted

Would you classify this as a bug in AAPT? I believe it may have good reason the way it is.